src/common/MuxLogger.cpp

```diff
@@ -99,12 +100,13 @@ void MuxLogger::initialize(
     boost::log::settings settings;
     boost::log::init_from_settings(settings);
 
-//    boost::filesystem::remove(path);
-
-//    boost::log::add_file_log(
-//        keywords::file_name = path,
-//        keywords::format = "[%TimeStamp%] [%Severity%] %Message%"
-//    );
+    if (extraLogFile) {
+        boost::filesystem::remove(path);
```

Comment:
Do we want to add some protection on this `remove()`? A try/catch maybe?

Reply:
If the file doesn't exist, `remove` will return `false`, for those exceptions that `remove` might throw(rw issue, OS error), we don't have to catch them, right?

Reply:
I guess it's fine, I assume we are using this argument only for dev/debug purpose.